src/database/DataTypes/Item.js

```diff
@@ -246,6 +246,7 @@ Item.schema = {
     isVisible: { type: 'bool', default: false },
     crossReferenceItem: { type: 'Item', optional: true },
     unit: { type: 'Unit', optional: true },
+    defaultRestrictedLocationType: { type: 'LocationType', optional: true },
```

Comment:
I'm hoping that the meaning of a `restrictedLocationType` will become clear as I read on..